docs/UserGuide.md

```diff
@@ -3,176 +3,203 @@ layout: page
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Cena m/A0123456J e/thechamp@u.nus.edu`
+* `add n/Alvina Handsome m/A0123456X e/handsome.alvina@u.nus.edu t/2G`
 
-### Listing all persons : `list`
+### <a name="list"></a>Listing all students : `list`
 
-Shows a list of all persons in the address book.
+Shows a list of all students in the current class list.
 
-Format: `list`
+```
+list
+```
 
-### Editing a person : `edit`
+### <a name="find"></a>Locating students by name: `find`
 
-Edits an existing person in the address book.
+Finds students whose names contain any of the given keywords.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+```
+find KEYWORD [MORE_KEYWORDS]
+````
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* The search is case-insensitive. e.g `john` will match `John`
+* The order of the keywords does not matter. e.g. `John Cena` will match `Cena John`
+* Only the name is searched.
+* Only full words will be matched e.g. `Jon` will not match `John` 
+* Students matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `John Cena` will return `John Tan`, `Mark Cena`
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `find John` returns `john` and `John Cena`
+* `find kent ridge` returns `Kent Tan` and `Mark Ridge`<br>
+[Insert image of an example of result of `find Cena`]
 
-### Locating persons by name: `find`
+### <a name="delete"></a>Deleting a student : `delete`
 
-Finds persons whose names contain any of the given keywords.
+The user will be prompted to confirm their decision here.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+If yes: deletes the specified student from the class list.
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+```
+delete INDEX
+```
+
+* Deletes the person at the specified `INDEX`.
```

Comment:
Perhaps "person" can be changed to "student"? I noticed the same issue in several other places too.